scalabel/eval/mot.py

```diff
@@ -42,6 +41,44 @@
     "mostly_lost": "ML",
     "num_fragmentations": "FM",
 }
+METRIC_TO_AVERAGE = set(["IDF1", "MOTA", "MOTP"])
+
+
+class BoxTrackResult(Result):
+    """The class for bounding box tracking evaluation results."""
+
+    mMOTA: float
```

Comment:
why do we still have mMOTA etc? This should correspond to the top-level category AVERAGE? So why store it separately again?